templates/repo/issue/view_content/comments.tmpl

```diff
@@ -371,27 +371,31 @@
 		{{else if eq .Type 22}}
 			<div class="timeline-item-group" id="{{.HashTag}}">
 				<div class="timeline-item event">
+					{{$reviewType := -1}}
```

Comment:
Add comment why we set this value?

Reply:
I think it has been explained by the next line: `{{if .Review}}{{$reviewType = ...}}{{end}}`

Reply:
Okay so it's just a init value with no special meaning, right?

Reply:
No special meaning. `-1` is chosen to avoid conflicting with some default value `0`.